libs/application/ui-components/src/components/ApplicationList/ApplicationList.tsx

```diff
@@ -64,10 +66,25 @@ const DefaultData: Record<ApplicationStatus, DefaultStateData> = {
       label: coreMessages.cardButtonInProgress,
     },
   },
+  [ApplicationStatus.NOT_STARTED]: {
```

Comment:
@alexdiljar

Reply:
Gunnar has been is on holiday right now I think. This will not effect other applications and we ran this by Vésteinn and Daði our designer. We need this functionality for this application. I'll discuss this with Gunnar once he gets back and we can reevaluate it then, and I think a wider discussion about how the application system can support applications that are waiting for users might be needed. I'll be sure to bring it up on the next umsóknarkefis faghópur meating.